src/scripts/shell.js

```diff
@@ -1,20 +1,53 @@
 // TODO: This seems like a good candidate for deprecation
 export default {
-    openUrl: function (url, target) {
+    enableFullscreen: function() {
+        if (window.NativeShell) {
+            window.NativeShell.enableFullscreen();
```

Comment:
This looks like a good place to use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) `?.`.

Reply:
Also, the diff would be much easier to read if you move the fullscreen methods back below the URL one, but I suppose it doesn't matter much.